CODEOWNERS

```diff
@@ -0,0 +1,70 @@
+# This is a comment.
```

Comment:
replace with motivation of why we are doing this... something like

MoveIt! is a large project with pull requests being created often. In order to ensure quick review turn around time from our maintainer team, we're leveraging an automated "triage" approach to auto-assign reviews to new pull requests. If you already know who should review your PR, then you can assign them manually. Otherwise, this code owners file will help assign reviewers and reduce the amount of email received by maintainers having to "Watch" the entire repository.

Reply:
Might lower the barrier of entry and clarify expectations to add a line like this:
"Code owners are maintainers who are familiar with certain parts of the code, and thus more likely to review a PR relating to them. Being a code owner does not imply authorship, control or ownership in a legal sense."

Reply:
Done